### PR TITLE
fix esg link doesn't get send to client

### PR DESCRIPTION
**Pull request by leangseu-edx**
**TL;DR -** 

JIRA: [JIRA-XXXX](https://openedx.atlassian.net/browse/JIRA-XXXX)

**What changed?**

Only 2 files changed:
- [openassessment/xblock/openassessmentblock.py](https://github.com/openedx/edx-ora2/pull/1786/files?file-filters%5B%5D=.js&file-filters%5B%5D=.py&show-viewed-files=true#r789652414)
- [openassessment/xblock/static/js/src/lms/oa_course_items_listing.js](https://github.com/openedx/edx-ora2/pull/1786/files?file-filters%5B%5D=.js&file-filters%5B%5D=.py&show-viewed-files=true#r789654721)

The rest of the files are translation and generated static.

**Developer Checklist**

- [ ] Reviewed the [release process](https://github.com/edx/edx-ora2/blob/master/.github/release_process.md)
- [ ] Translations and JS/SASS compiled
- [ ] Bumped version number in [setup.py](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/setup.py#L39) and [package.json](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/package.json#L3)

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [ ] I've done a visual code review
- [ ] I've tested the new functionality

FYI: @edx/masters-devs-gta

### CODE REVIEW

File: openassessment/xblock/openassessmentblock.py [hunk 1]
```diff
@@ -588,7 +588,8 @@ def ora_blocks_listing_view(self, context=None):
             additional_css=["static/css/lib/backgrid/backgrid%s.css" % min_postfix],
             additional_js=["static/js/lib/backgrid/backgrid%s.js" % min_postfix],
             additional_js_context={
-                "ENHANCED_STAFF_GRADER": self.is_enhanced_staff_grader_enabled
+                "ENHANCED_STAFF_GRADER": self.is_enhanced_staff_grader_enabled,
+                "ORA_GRADING_MICROFRONTEND_URL": settings.ORA_GRADING_MICROFRONTEND_URL
```
Comment (leangseu-edx) [inline openassessment/xblock/openassessmentblock.py]: This line was lost during rebase which is why the ora grading link was incorrect.

File: openassessment/xblock/static/js/src/lms/oa_course_items_listing.js [hunk 2]
```diff
@@ -144,7 +144,7 @@ export class CourseItemsListingView {
       },
     ];
 
-    if (esgEnabled) {
+    if (itemViewEnabled && esgEnabled) {
```
Comment (leangseu-edx) [inline openassessment/xblock/static/js/src/lms/oa_course_items_listing.js]: make sure response column doesn't show up if item view is disabled

File: openassessment/xblock/static/js/src/lms/oa_course_items_listing.js [hunk 3]
```diff
@@ -34,7 +34,7 @@ export class CourseItemsListingView {
         let link = null;
         if (itemViewEnabled && (!this.type || (this.type && hasAssessmentType))) {
           const displayValue = this.title || formattedValue;
-          if (esgEnabled && esgRootUrl) {
+          if (esgEnabled) {
```
Comment (leangseu-edx) [inline openassessment/xblock/static/js/src/lms/oa_course_items_listing.js]: stop checking root url. It is unnecessary confusing when error occur.

**Review by nsprenkle — APPROVED** [review]
:shipit:

**Review by jansenk — APPROVED** [review]
